taxi/urls.py

```diff
@@ -6,22 +6,25 @@
     CarDetailView,
     DriverListView,
     DriverDetailView,
-    ManufacturerListView,
+    ManufacturerListView, ManufacturerCreateView, ManufacturerUpdateView, ManufacturerDeleteView, CarCreateView,
+    CarUpdateView, CarDeleteView,
 )
 
 urlpatterns = [
     path("", index, name="index"),
-    path(
-        "manufacturers/",
-        ManufacturerListView.as_view(),
-        name="manufacturer-list",
-    ),
+    path("manufacturers/", ManufacturerListView.as_view(), name="manufacturer-list", ),
+    path("manufacturers/create/", ManufacturerCreateView.as_view(), name="manufacturer-create"),
+    path("manufacturers/<int:pk>/update/", ManufacturerUpdateView.as_view(), name="manufacturer-update"),
+    path("manufacturers/<int:pk>/delete/", ManufacturerDeleteView.as_view(), name="manufacturer-delete"),
+
     path("cars/", CarListView.as_view(), name="car-list"),
     path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
+    path("cars/create/", CarCreateView.as_view(), name="car-create"),
+    path("cars/<int:pk>/updade/", CarUpdateView.as_view(), name="car-update"),
```

Comment:
There is a typo in the URL pattern for updating a car. The path should be 'update/' instead of 'updade/'. This will cause a routing error when trying to access the car update view.